src/Test/Utilities/Portable/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -181,6 +184,11 @@ public static string GetFlowGraph(Compilation compilation, ControlFlowGraph grap
             regionMap.Free();
             return pooledBuilder.ToStringAndFree();
 
+            string getDestinationString(ref BasicBlock.Branch branch)
+            {
+                return branch.Destination != null ? ("B" + branch.Destination.Ordinal) : "null";
```

Comment:
I'd missed the PR where you added the 'B' prefix to branch numbers. Thanks a lot! That definitely helps with reading the tests.